examples/parcel/.compiledcssrc

```diff
@@ -1,3 +1,10 @@
 {
-  "importReact": false
+  "importReact": false,
+  "extensions": [
```

Comment:
Just an idea, but... instead of providing extensions in the config here, you could be a bit more clever in parcel and webpack. What you really want to know is which extensions are used with the `@compiled/parcel-transformer` or the `@compiled/webpack-loader`. I'm not sure how easy it would be to codify that though.

Reply:
yeah, I had a quick look into that before implementing the config and ended up going this route because it got complex very quick, as I'd have to programmatically get the matcher/regex.

I'll have another pass to see if I can figure out an elegant solution otherwise will create an issue to track this.

Reply:
Created an issue for it as I couldn't find an elegant way to address this without refactoring how we build the resolver https://github.com/atlassian-labs/compiled/issues/1000